moviepy/video/VideoClip.py

```diff
@@ -323,6 +323,7 @@ def write_videofile(
                 bitrate=audio_bitrate,
                 write_logfile=write_logfile,
                 verbose=verbose,
+                ffmpeg_params=ffmpeg_params,  
```

Comment:
I'm wondering if wouldn't be desirable to allow different parameters for audio and video

Reply:
I mean, add an argument `audio_ffmpeg_params=None` to the method and pass that value here. It'd allow more granularity on the custom ffmpeg behaviour, right?

Reply:
Yes, definetively.